chapters/packages.tex

```diff
@@ -321,3 +321,134 @@ \section{External Resources}\label{external-resources}
 \filename{modelica://Modelica/Mechanics/C.jpg} is illegal.
 The references \filename{modelica://Modelica.Mechanics/C.jpg} and \filename{modelica://Modelica/C.jpg} must also refer to two distinct resources.
 \end{example}
+
+\section{Multilingual Descriptions}\label{multilingual-descriptions}
+
+\begin{nonnormative}
+Descriptive texts in a model or library are usually formulated in English.
+To support users who are not so familiar with this, a Modelica tool can support the translation of these texts into any language.
+Translated Modelica text is provided by external files, so that no modification of the original Modelica text is required.
+\end{nonnormative}
+
+The texts in following Modelica constructs should be translated:
+\begin{itemize}
+\item description strings of component declarations and classes
+\item strings in the following annotations:
+\begin{itemize}
+\item \lstinline!Text.string!, \lstinline!Text.textString!
+\item \lstinline!missingInnerMessage!, \lstinline!obsolete!, \lstinline!unassignedMessage!
+\item \lstinline!Dialog.group!, \lstinline!Dialog.tab!
+\item \lstinline!Dialog.loadSelector.caption!, \lstinline!Dialog.loadSelector.filter!, \lstinline!Dialog.saveSelector.caption!, \lstinline!Dialog.saveSelector.filter!
+\item \lstinline!Documentation.info!, \lstinline!Documentation.revisions!
+\item \lstinline!Figure.title!, \lstinline!Plot.title!, \lstinline!Curve.legend!
+\end{itemize}
+\end{itemize}
+
+C-style and line comments are not translated.
+
+The files to support translation must be provided along with the library.
+They must be stored in the resources directory \filename{modelica://$\mathit{LibraryName}$/Resources/Language/}.
+
+Two kind of files have to be provided:
+\begin{enumerate}
+\item Template file \filename{$\mathit{LibraryName}$.pot}, as the resource \filename{modelica://$\mathit{LibraryName}$/Resources/Language/$\mathit{LibraryName}$.pot}.
+It contains all necessary information to translate all descriptions, but no translations.
```

Comment:
```suggestion
It contains all necessary information to translate the texts in the library, but no translations.
```
Spell-checker didn't like "translateable" - and it is kind of redundant.

However, I deliberately changed to "the texts" - instead of "all texts"; one reason is that if a library wants to translate only some strings that should still work - another is that there are some texts (like comments) that we don't support translation for.

Reply:
… but it turns out that you actually speak of _strings_ when explaining what you did:
```suggestion
It contains all necessary information to translate strings in the library, but no translations.
```

Reply:
I'm a bit unsure if it should be "the strings" or "strings".

Reply:
I also thought of _the strings_, but to me it sounded indistinguishable from _all strings_.

Reply:
As suggested in the meeting:
```suggestion
It describes all translatable strings in the library, but does not contain any translations.
```